spec/support/common_client_stubs.rb

```diff
@@ -0,0 +1,39 @@
+shared_context :stub_api_following_standards do
```

Comment:
Do we expect other common stubs to go in here, or is this likely to be the only one? If this is likely to be the only one, we might want to switch the file name to match the shared context name.